### PR TITLE
Nerpa devnet

### PR DESCRIPTION
**Pull request by pooja**
In this PR:
- Specify how to connect to Nerpa devnet in interacting with the network guide
- Call this info out more clearly in the "Install Filecoin" section
- Add links to codebase and demo video for the Meme Marketplace and Simple Pinning Service examples

### CODE REVIEW

File: docs/build/start-building/interacting-with-the-network.md [hunk 1]
```diff
@@ -28,20 +28,20 @@ Here are some scripts to set up your own remote networks for development:
...
 
-To connect to the Devnet:
+To connect to the [Nerpa Devnet](http://www.nerpa.fildev.network/):
 
 1. Delete any existing local Lotus repository, if one is present.
-2. Build and run Lotus from branch `interopnet (bead3bf)`, `tag interop.6.16.0`. This should automatically connect you to the proper bootstrappers.
-3. Once running, several miners are configured to auto-accept any proposed deals.
+2. Build and run Lotus from branch `ntwk-nerpa`, [`tag ntwk-nerpa-7.7.0`](https://github.com/filecoin-project/lotus/tree/ntwk-nerpa-7.7.0). When you run `lotus daemon` on the build from this branch, you should automatically be connected to the proper bootstrappers.
+3. There should be a few miners on the network that are configured to accept storage deals. If you are having trouble finding a miner to accept your storage deal, please ping for support in the #hfs-sponsor-filecoin-team channel on the [Filecoin slack](https://filecoin.io/slack).
```
Comment (mishmosh) [inline docs/build/start-building/interacting-with-the-network.md:37]: This is a private channel that I believe only registered HackFS participants are invited to, so the docs should point people somewhere else. Maybe #fil-storage-dev?

File: docs/build/start-building/interacting-with-the-network.md [hunk 2]
```diff
@@ -28,20 +28,20 @@ Here are some scripts to set up your own remote networks for development:
 
 ## Devnet
 
-In order to make testing easier, a live devnet (developer network) is now available for use. It features 512MB sectors and reduced proofs parameters, so sealing is much faster than in the full Testnet version.
+In order to make testing easier, a live devnet (developer network) is now available for use. We call this network the Nerpa Devnet (the nerpa is a species of seal, and one of the smallest true seals). The Nerpa Devnet features 512MB sectors (in addition to 32GB and 64GB sectors) and reduced proofs parameters, so sealing is much faster than in the full Testnet version. This devnet is currently primarily intended for participants in the [HackFS hackathon](https://hackfs.com).
```
Comment (mishmosh) [inline docs/build/start-building/interacting-with-the-network.md:31]: This is intended to be a general-use devnet. All devgrant winners etc are also being pointed here. Can we delete the last sentence in 31?

File: docs/build/start-building/interacting-with-the-network.md (hunk 1 above)
Comment (AndrewDiMola) [reply docs/build/start-building/interacting-with-the-network.md:37]: See #135.

File: docs/build/start-building/interacting-with-the-network.md (hunk 2 above)
Comment (AndrewDiMola) [reply docs/build/start-building/interacting-with-the-network.md:31]: See #135.

**Pull request merged**